### DISCUSSION TITLE
Can't make Redshift source work

### DISCUSSION THREAD

**Discussion by jdenquin** [grouparoo/grouparoo#1139] [Q&A] [answered]
Hello There!

I'm currently trying Grouparoo and it seems great, but I've a problem connecting my RedShift warehouse to it.
I put my host, port, database and credentials, but when I make a connection's test, it says that no tables were found in the database. I've many tables into it.
So I can't create a source, maybe can you help me with that?

Regards,
Jeremy

**Comment by bleonard** [comment, marked as answer]
It sounds like it is connecting if there is a green message but with 0 tables.
Maybe there is a schema? The default is "public"
There are also possible access level (IAM) issues that happen frequently in Amazon. Does that user have [access](https://chartio.com/learn/amazon-redshift/how-to-view-permissions-in-amazon-redshift/) to all the tables?

To see what tables it has, Grouparoo does a query like this:
`select count(*) as __count from information_schema.tables where table_schema = 'public'`
What does that give you outside of Grouparoo?

**Reply by jdenquin** [reply]
Empty! But I finally found the solution, it appears that my different tables are under a different schema, not easy to understand, I'm a newbie with Redshift!

Btw thanks for your fast answer @bleonard!
